common/types/src/runtime.proto

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+package kyve.tendermint;
```

Comment:
Are you thinking of creating one `.proto` file for every runtime parent class? Meaning all the EVM runtimes will use X `.proto` as a source, all Cosmos runtime will use `Y` as a source?

Is it possible to create just a single `.proto` file for all runtimes?

Reply:
it should not be here, it was a leftover from when i did multiple implementations to compare them